readthedocs/doc_builder/environments.py

```diff
@@ -421,7 +421,11 @@ def update_build(self, state=None):
                 self.build['error'] = str(self.failure)
             else:
                 self.build['error'] = ugettext_noop(
-                    "An unexpected error occurred")
+                    "A failure in our code has occured. "
```

Comment:
Nitpicks because apparently this is very commonly seen copy:

* "failure" -> "error": failure is a state we apply, error is what we hit
* "in our code" is very specific, where this can be any number of things we did. something more generic would be good here

Others:

"There was a problem with Read the Docs while building your documentation"
"We encountered an error with Read the Docs while building your documentation"